www/src/layouts/getting-started.js

```diff
@@ -0,0 +1,28 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import PageHeader from '../components/PageHeader';
+import Main from '../components/Main';
+import Default from './index';
+
+const propTypes = {
+  location: PropTypes.object.isRequired,
+};
+
+function DocPage({ children, ...props }) {
+  return (
+    <Default {...props}>
+      <PageHeader
+        title="Getting started"
+        subTitle={''}
```

Comment:
`subTitle=""`

Reply:
actually,

1. `<PageHeader>` probably shouldn't use `React.createClass`
2. The "default subtitle to empty" logic as applicable should probably live in `<PageHeader>` (actually it seems like it can already deal just fine with null-y subtitles?